### ISSUE TITLE
Write blog post about PDF accessibility for math formulas

### ISSUE THREAD

**Issue by ronaldtse** [metanorma/metanorma.org#467]
The BIPM SI Brochure utilizes several advanced PDF techniques for accessibility, not only using PDF/UA features but also making math formulas accessible for non-Adobe PDF viewers.

They include:
* Embedding MathML formulas as attachments
* techniques to render AsciiMath/LaTeX Math as PDF content to allow copy/pasting for non-Adobe readers (PDF Content tree)
* Using PDF/UA features such as "Actual Text" and "Alternate Text" for tag readers (PDF Tag tree)

_Originally posted by @ronaldtse in https://github.com/metanorma/bipm-si-brochure/issues/118#issuecomment-901528414_

**Comment by ronaldtse** [comment]
https://answers.acrobatusers.com/What-difference-title-actual-text-alternative-text-figure-properties-image-q16218.aspx

> From an Adobe document about Acrobat and accessible PDF — 
> • Title – This is the identifying text that will appear next to the element tag in the logical structure tree.
> 
> • Type – This defines the element tag type that will appear in the logical structure tree. For best results, it is suggested that you change only to one of the Adobe standard element types.
> 
> • Actual Text – This is the actual text that will be sent to a screen reader. If text is entered, the entered text will be read, and not the text that may comprise the displayed document content. This property should be entered only if you want something other than the content of an element to be read by a screen reader. Basically, this is replacement text.
> 
> • Alternate Text – This is additional or descriptive text that can be used to describe an image, formula, or other item in the document that does not translate naturally into text. For example, a pie chart may be described giving the values of the various sectors to provide users with visual impairments with more detailed information than generally available from a simple caption.
> 
> 
> Also, in the Content tab of the Properties dialog there is the"Expansion Text" field. 
> This is where you expand an abbreviation, acronym or "jargon".
> 
> Discussion in ISO 32000-1:
> Section 14.9.3 for Alternate Descriptions ("Alt Text")
> Section 14.9.4 Replacement Text ("Actual Text") 
> Section 14.9.5 Expansion of Abbreviations and Acronyms 
> For an ISO approved copy of the ISO 32000-1 Standards document: 
> http://wwwimages.adobe.com/www.adobe.com/content/dam/Adobe/en/devnet/pdf/pdfs/P
> DF32000_2008.pdf
>

**Comment by ronaldtse** [comment]
This has been done a while ago.
